src/UglyToad.PdfPig/Graphics/Colors/ColorSpaceExtensions.cs

```diff
@@ -36,20 +36,21 @@ public static ColorSpaceFamily GetFamily(this ColorSpace colorSpace)
 
         /// <summary>
         /// Maps from a <see cref="NameToken"/> to the corresponding <see cref="ColorSpace"/> if one exists.
+        /// <para>Includes extended color spaces.</para>
         /// </summary>
         public static bool TryMapToColorSpace(this NameToken name, out ColorSpace colorspace)
         {
             colorspace = ColorSpace.DeviceGray;
 
-            if (name.Data == NameToken.Devicegray.Data)
+            if (name.Data == NameToken.Devicegray.Data || name.Data == "G")
             {
                 colorspace = ColorSpace.DeviceGray;
             }
-            else if (name.Data == NameToken.Devicergb.Data)
+            else if (name.Data == NameToken.Devicergb.Data || name.Data == "RGB")
```

Comment:
Given how few documents actually follow the spec would it make sense to do a case insensitive check in this file, e.g.

```
string.Equals(name.Data, "rgb", StringComparison.OrdinalIgnoreCase)
```

Reply:
Hehe I agree with you on the idea but it seems we're safe on this as this is how it was done even before the change (see here src/UglyToad.PdfPig/Util/ColorSpaceDetailsParser.cs)

I'll have a look in my next PR